test/mocha.opts

```diff
@@ -1 +1,3 @@
+--compilers ts-node/register
```

Comment:
`--compilers` is deprecated in mocha: https://github.com/mochajs/mocha/wiki/compilers-deprecation

You can just do `--require` like the other two.